examples/go.mod

```diff
@@ -2,11 +2,12 @@ module github.com/fullstorydev/emulators/examples
 
 go 1.18
 
+toolchain go1.21.6
```

Comment:
🤔 The `toolchain` is still being automatically added?

Reply:
No, I think I messed up.  Fifth time's a charm?